### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by hargata** [redimp/otterwiki#102] [General]
Hey, just wanted to say that this is a great piece of software. We use it for our documentation over at https://docs.lubelogger.com

Otterwiki is very minimal and does exactly what we need it to do. 

As a dev, I know that sometimes it feels like the only time your users reach out to you is to complain or make requests, well this is just to tell you how grateful we are that you made Otterwiki. It's good software, keep it up.

Once again, thank you.

**Comment by redimp** [comment]
You are very welcome. I appreciate your feedback! 

I am glad that this project is helpful to others.

**Comment by xgpt** [comment]
I was actually directed to your project by the wonderful Bookstack project. If I was planning on collaboration, I would use bookstack, however, as a single-user I'm ecstatic to introduce otterwiki into my own routines. I'm very excited by the flat file database, the git integration, and ease of scriptable backups without learning SQL (again). I used to use dokuwiki but the syntax for that is a mess. I've been using Obsidian but that's been very bloated for my simplistic needs, and the syncing has been a bit of a nightmare as well. (using syncthing)

Point is, this is *exactly* what I've been looking for. Thank you!

**Comment by Gamedirection** [comment]
I just wanted to take a moment to express my gratitude for Otter Wiki. What started as curiosity (thanks to the adorable otter!) turned into genuine admiration for the project’s clean workflow and thoughtful design. It’s rare to find tools that balance simplicity with such powerful utility—your work stands out.   

Thank you, too, for the patient support and dedication to the community. Projects like this make a real difference, and I’m thrilled to be using it. Keep up the fantastic work!

**Comment by Allexz1** [comment]
Wow, what a product! Thank you so much!

I realized bookstacks just wasnt for me when it decided to break during updates.... so today i spent 6hours googling, reading, searching, spinning up dokuwiki which i havent seen for nearly 10 years, wiki.js etc... even begging some llm and then there it was. 

The otter one... =)

Plain storage for crucial information is so very important which i thankfully again, today was reminded of.
It's projects like this that keep reminding me that i wish i knew coding, clean lean and awesome!

Thanks! <3

**Comment by Duplicake-fyi** [comment]
Also came here to say thank you. For my purposes at least, this is the best wiki software by far.

It's so fast, simple, and I love the UI. Thank you for making this amazing software!

**Comment by trailjeep** [comment]
I know this is old but... I have to jump in here because I too love Otter Wiki!

I started using it after bouncing around between Obsidian and Trilium.  Each has pluses and minuses, but Otter Wiki is best of both IMO (plain text / http access / and git backed).

Even better, after setting up [Hermes Agent](https://hermes-agent.nousresearch.com/) and learning about [LLM Wiki](https://gist.github.com/karpathy/442a6bf555914893e9891c11519de94f) I set it up with Otter Wiki and will never look back - the git server simplifies everything.

Only thing missing is graph view - never used/cared for it anyway.

Just for kicks asked I 
> am i missing any functionality (not sure that's best term) by using otterwiki for llm wiki backend instead of obsidian?

Excerpts:

> Your Obsidian question — verdict: for an LLM-wiki, nothing essential. The gaps are desktop-app conveniences, not second-brain functionality. What you'd trade away is worse than what you'd gain.

> Bottom line: keep OtterWiki — it's the right backend for an agent-operated brain. Your vault is plain markdown in git, so if you ever crave a graph or offline reading, you can point a desktop tool at the repo read-only without migrating anything.

Thank You!
